clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -9171,6 +9171,9 @@ static void ProcessDeclAttribute(Sema &S, Scope *scope, Decl *D,
   case ParsedAttr::AT_SYCLDeviceIndirectlyCallable:
     handleSYCLDeviceIndirectlyCallableAttr(S, D, AL);
     break;
+  case ParsedAttr::AT_SYCLGlobalVar:
+    handleSimpleAttribute<SYCLGlobalVarAttr>(S, D, AL);
```

Comment:
This code should be removed. You should add `let SimpleHandler = 1;` in Attr.td instead.